packages/office-ui-fabric-react/src/components/Modal/examples/Modal.Basic.Example.tsx

```diff
@@ -2,99 +2,108 @@ import * as React from 'react';
-              sagittis convallis ante. Phasellus malesuada eleifend mollis. Curabitur ultricies leo ac metus venenatis elementum.{' '}
-            </p>
-            <p>
-              Aenean egestas quam ut erat commodo blandit. Mauris ante nisl, pellentesque sed venenatis nec, aliquet sit amet enim. Praesent
-              vitae diam non diam aliquet tristique non ut arcu. Pellentesque et ultrices eros. Fusce diam metus, mattis eu luctus nec,
-              facilisis vel erat. Nam a lacus quis tellus gravida euismod. Nulla sed sem eget tortor cursus interdum. Sed vehicula tristique
-              ultricies. Aenean libero purus, mollis quis massa quis, eleifend dictum massa. Fusce eu sapien sit amet odio lacinia placerat.
-              Mauris varius risus sed aliquet cursus. Aenean lectus magna, tincidunt sit amet sodales a, volutpat ac leo. Morbi nisl sapien,
-              tincidunt sit amet mauris quis, sollicitudin auctor est.{' '}
-            </p>
-            <p>
-              Nam id mi justo. Nam vehicula vulputate augue, ac pretium enim rutrum ultricies. Sed aliquet accumsan varius. Quisque ac
-              auctor ligula. Fusce fringilla, odio et dignissim iaculis, est lacus ultrices risus, vitae condimentum enim urna eu nunc. In
-              risus est, mattis non suscipit at, mattis ut ante. Maecenas consectetur urna vel erat maximus, non molestie massa consequat.
-              Duis a feugiat nibh. Sed a hendrerit diam, a mattis est. In augue dolor, faucibus vel metus at, convallis rhoncus dui.
-            </p>
-          </div>
-        </Modal>
-      </div>
-    );
-  }
-
-  private _showModal = (): void => {
-    this.setState({ showModal: true });
-  };
-
-  private _closeModal = (): void => {
-    this.setState({ showModal: false });
-  };
-
-  private _toggleDraggable = (): void => {
-    this.setState({ isDraggable: !this.state.isDraggable });
-  };
-}
+  return (
+    <div>
+      <Toggle label="Is draggable" inlineLabel onChange={() => setIsDraggable(!isDraggable)} checked={isDraggable} />
+      <DefaultButton secondaryText="Opens the Sample Modal" onClick={() => setShowModal(true)} text="Open Modal" />
+      <Modal
+        titleAriaId={titleId}
+        subtitleAriaId={subtitleId}
+        isOpen={showModal}
+        onDismiss={() => setShowModal(false)}
+        isBlocking={false}
+        containerClassName={classNames.container}
+        dragOptions={isDraggable ? dragOptions : undefined}
+      >
+        <div className={classNames.header}>
+          <span id={titleId}>Lorem Ipsum</span>
+        </div>
+        <div id={subtitleId} className={classNames.body}>
+          <DefaultButton onClick={() => setShowModal(false)} text="Close" />
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas lorem nulla, malesuada ut sagittis sit amet, vulputate in leo.
```

Comment:
Note that the excessive lorem ipsum changes are due to Prettier re-wrapping the text... 🤦‍♀ 
 (and in general, this change is best viewed with "ignore whitespace changes")

Reply:
don't we have a lorem generator?